lib/plain/entities/ai-action.ts

```diff
@@ -65,13 +65,12 @@ export type AiActionPlainClientAPI = {
   /**
    * Publishes the AI Action.
    * @param params Entity IDs to identify the AI Action.
-   * @param payload The AI Action details.
    * @param headers Optional headers for the request.
    * @returns The published AI Action and its metadata.
    * @throws if the request fails or the payload is malformed.
    */
   publish(
-    params: OptionalDefaults<GetSpaceParams & { aiActionId: string }>,
+    params: OptionalDefaults<GetSpaceParams & { aiActionId: string; version: number }>,
     payload: AiActionProps,
```

Comment:
We just want to have the argument order right. We don't expect them to pass in the ai action again.

```suggestion
    payload?: unknown,
```